stock_location_flowable/i18n/es.po

```diff
@@ -0,0 +1,382 @@
+#: code:addons/stock_location_flowable/models/stock_location.py:0
+#: model:ir.actions.act_window,name:stock_location_flowable.action_picking_tree_blocked
+#: model_terms:ir.ui.view,arch_db:stock_location_flowable.stock_picking_type_kanban
+#, python-format
+msgid "Blocked"
+msgstr "Bloqueado"
+
+#. module: stock_location_flowable
+#: model:ir.model.fields,field_description:stock_location_flowable.field_stock_location__flowable_capacity
+#: model_terms:ir.ui.view,arch_db:stock_location_flowable.stock_location_view_form
+msgid "Capacity"
+msgstr "Capacidad"
+
+#. module: stock_location_flowable
+#: code:addons/stock_location_flowable/models/stock_location.py:0
+#, python-format
+msgid "Capacity must be greater than 0"
+msgstr "La capacidad debe ser mayor que 0."
+
+#. module: stock_location_flowable
+#: code:addons/stock_location_flowable/models/stock_location.py:0
+#, python-format
+msgid "Capacity must be greater than capacity occupied %s"
+msgstr "La capacidad debe ser mayor que la capacidad ocupada %s"
+
+#. module: stock_location_flowable
+#: model:ir.model.fields,field_description:stock_location_flowable.field_stock_picking_type__count_picking_blocked
+msgid "Count Picking Blocked"
+msgstr "Conteo de Albarán Bloqueado"
+
+#. module: stock_location_flowable
+#: model:ir.model.fields,field_description:stock_location_flowable.field_stock_location__create_lots
+#: model_terms:ir.ui.view,arch_db:stock_location_flowable.stock_location_view_form
+msgid "Create Lots"
+msgstr "Crear Lotes"
+
+#. module: stock_location_flowable
+#: model:ir.model.fields,field_description:stock_location_flowable.field_mrp_production__display_name
+#: model:ir.model.fields,field_description:stock_location_flowable.field_stock_location__display_name
+#: model:ir.model.fields,field_description:stock_location_flowable.field_stock_move__display_name
+#: model:ir.model.fields,field_description:stock_location_flowable.field_stock_move_line__display_name
+#: model:ir.model.fields,field_description:stock_location_flowable.field_stock_picking__display_name
+#: model:ir.model.fields,field_description:stock_location_flowable.field_stock_picking_type__display_name
+#: model:ir.model.fields,field_description:stock_location_flowable.field_stock_return_picking__display_name
+msgid "Display Name"
+msgstr "Nombre mostrado"
+
+#. module: stock_location_flowable
+#: model_terms:ir.ui.view,arch_db:stock_location_flowable.stock_location_view_form
+msgid "Enabled Storage"
+msgstr "Almacenamiento Habilitado"
+
+#. module: stock_location_flowable
+#: model_terms:ir.ui.view,arch_db:stock_location_flowable.stock_location_view_form
+msgid "Enabled"
+msgstr "Habilitado"
+
+#. module: stock_location_flowable
+#: model_terms:ir.ui.view,arch_db:stock_location_flowable.stock_location_view_form
+msgid "Flowable"
+msgstr "Fluido"
+
+#. module: stock_location_flowable
+#: model:ir.model.fields,field_description:stock_location_flowable.field_stock_location__flowable_blocked
+msgid "Flowable Blocked"
```

Comment:
Just "Blocked"
Don't forget to rename the same string from its corresponding field